website/README.md

```
@@ -0,0 +1,53 @@
+# Chapplin Documentation Website
+
+This is the official documentation website for Chapplin, built with [Astro](https://astro.build) and [Starlight](https://starlight.astro.build).
+
+[![Built with Starlight](https://astro.badg.es/v2/built-with-starlight/tiny.svg)](https://starlight.astro.build)
+
+## 🚀 Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)